src/AnalyzeView/ULogParser.cc

```diff
@@ -14,25 +14,25 @@ ULogParser::~ULogParser()
 
 int ULogParser::sizeOfType(QString& typeName)
 {
-    if (typeName == QLatin1Literal("int8_t") || typeName == QLatin1Literal("uint8_t")) {
```

Comment:
In most other parts of the code I've given up on stuff like this and just do:
```
if (typeName == "int8_t") || ty
```

It causes extra run time code to be run, but it has no real impact on performance and the readability of the code without it is way better. So I lean to readability as opposed to trying to reduce a performance hit you can't measure above microseconds.

Reply:
I just replaced deprecated API to new one. I did not diagnose it is necessary or not. So remove `QLatin1Literal` completelly? Only in this file or in `APMAirframeComponentController.cc` as well?

Reply:
It's all fine. Just letting you know for other changes you make. No need to go back and change these.

Reply:
ok, thanks, do we merging this? :slightly_smiling_face: